test/functional/feature_anchors.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test Anchors functionality"""
+
+from test_framework.p2p import P2PInterface
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+
+import os
```

Comment:
nit, per PEP8, this import should be before the framework ones (separated by a blank line)

Reply:
https://www.python.org/dev/peps/pep-0008/#imports

Imports should be grouped in the following order:

-   Standard library imports.
-    Related third party imports.
-    Local application/library specific imports.

You should put a blank line between each group of imports.

Reply:
Thanks, just fixed!